lib/auth/oidc.go

```diff
@@ -878,10 +888,10 @@ func (a *Server) getClaims(oidcClient *oidc.Client, connector services.OIDCConne
 			if !trace.IsNotFound(err) {
 				return nil, trace.Wrap(err)
 			}
-			log.Debugf("Found no GSuite claims.")
+			log.Debugf("Found no Google Workspace claims.")
 		} else {
 			if gsuiteClaims != nil {
-				log.Debugf("Got GSuiteclaims: %v.", gsuiteClaims)
+				log.Debugf("Got GoogleWorkspace claims: %v.", gsuiteClaims)
```

Comment:
This is the name of a variable so should be left the same.

```suggestion
				log.Debugf("Got gsuiteClaims: %v.", gsuiteClaims)
```

Reply:
@webvictim should we add " from Google Workspace"?  the debug would show to the customer so that would give some more context.

Reply:
Yes, that would work. I feel that `DEBUG` level errors are in general designed to be sent to support staff to help customers figure issues out (rather than customers doing it directly), but adding some additional context for people who might not know that Google Workspace used to be called G Suite is fine.